packages/popmotion-pose/docs/api/react/posed.md

```diff
@@ -37,6 +41,39 @@ const DraggableCircle = posed.circle({
 export default ({ radius }) => <DraggableCircle r={radius} />
 ```
 
+#### Calling `pose` with a React component
+
+`pose` is called directly, passing in [posed props](/pose/api/props):
```

Comment:
#### Existing components

Existing components can be converted to posed components by calling `posed` directly:

```javascript
const PosedComponent = posed(MyComponent)(poseProps)
```

For performance and layout calculations, React Pose requires a reference to the underlying DOM element. So, the component to be posed **must pass `hostRef` to the host element's `ref` prop**.

```javascript
const MyComponent = ({ hostRef }) => <div ref={hostRef} />

const PosedComponent = posed(MyComponent)({
  draggable: true
})

export default () => <PosedComponent pose={isOpen ? 'open' : 'closed'} />
```

For FLIP support in a `PoseGroup`, it **optionally** needs to pass on the `style` prop:

```javascript
const MyComponent = ({ hostRef, style }) => <div ref={hostRef} style={style} />
```

Reply:
Excellently worded, thanks!